### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by QueenterJuma**
## What does this PR do?
This PR is raised to create a Portfolio using a Mobile template from Figma
##Description of the task to be completed
- [x] Build a personal portfolio site.
- [x] Use a mobile Figma design.
- [x] Flexbox to place elements in the
- [x] Use images and backgrounds to enhance the look of the website
– [x] Use the [HTML&CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css)
- [x] Build the middle section of the mobile website

## How should it be tested
Clone the repository
```
git clone https://github.com/QueenterJuma/Portfolio_1/pulls /2
```
Navigate to the folder
```
cd repo-name
```
Checkout the branch
```
git checkout branch_mobile-app
```

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,6 +68,145 @@ <h1>
...
+        <button>See Project</button>
+      </div>
+    </div>
+    <div class="card two">
+      <div class="plc-img2"></div>
+      <div class="card-text">
+        <h3>Multipost Stories</h3>
+        <ul class="prot">
+          <li><a href="">CANOPY</a></li>
+          <li><a href="">BACK END DEV</a></li>
```
Comment (jerryowusu) [inline index.html:97]: The styling seems not to be working effectively as shown in the images below. Kindly make sure everything is working fine. The container of the projects should be aligned with grid and has the proper background image.

Screenshot from 2022-10-29 23-15-59

File: index.html [hunk 2]
```diff
@@ -68,6 +68,145 @@ <h1>
...
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang">
+          <li><a href="">HTML</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">JAVASCRIPT</a></li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </div>
```
Comment (jerryowusu) [inline index.html:90]: Each card should be aligned with flex box.

File: index.html [hunk 3]
```diff
@@ -68,6 +68,145 @@ <h1>
         </div>
       </div>
     </section>
+
+    <section id="portfolio">
+    <div class="card one">
+      <div class="plc-img1" ></div>
```
Comment (jerryowusu) [inline index.html:74]: Kindly make sure that the project images are all working well with the right size and position.

File: index.html [hunk 4]
```diff
@@ -68,6 +68,145 @@ <h1>
...
+          <li><a href="">BACKEND DEV</a></li>
+          <li><a href="">2015</a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang">
+          <li><a href="">HTML</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">JAVASCRIPT</a></li>
+        </ul>
+        <button>See Project</button>
```
Comment (jerryowusu) [inline index.html:88]: The button should be a link a or button tag, and kindly follow the button styling: Active, Hover and Pressed (see design)

File: index.html [hunk 5]
```diff
@@ -68,6 +68,145 @@ <h1>
...
+        <ul class='prot'>
+          <li><a href="">CANOPY</a></li>
+          <li><a href="">BACKEND DEV</a></li>
+          <li><a href="">2015</a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang">
+          <li><a href="">HTML</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">JAVASCRIPT</a></li>
```
Comment (jerryowusu) [inline index.html:86]: Kindly ensure that the section has the margins (top, bottom, left, right) defined in the design,
the section should also use the font and colours (background and font) defined for the template.
Your design is currently different as shown below.

Screenshot from 2022-10-29 23-15-59

File: index.html [hunk 6]
```diff
@@ -68,6 +68,145 @@ <h1>
...
+          <ul>
+            <li>
+              <a href="https://github.com/QueenterJuma"
+                ><i class="fa-brands fa-github"></i
+              ></a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/in/queenter-juma-54a0b1222/"
+                ><i class="fa-brands fa-linkedin"></i
+              ></a>
```
Comment (jerryowusu) [inline index.html:170]: Kindly ensure that you use the correct background image, fonts, colors, and other dimensions in order for your design to conform with the figma design. 

Your Design

Screenshot from 2022-10-29 23-16-11

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @QueenterJuma 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional :sunglasses:

soclose


## Highlights
:heavy_check_mark: All linters are passing
:heavy_check_mark: Followed GItHub flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -68,6 +68,153 @@ <h1>
...
+      </div>
+    </div>
+    <div class="d-flex-column card two">
+      <div class="plc-img2"></div>
+      <div class="card-text">
+        <h3>Multipost Stories</h3>
+        <ul class="prot">
+          <li><a href="">CANOPY</a></li>
+          <li><a href="">BACK END DEV</a></li>
+          <li><a href="">2015</a></li>
```
Comment (christianbiring1) [inline index.html:98]: - [ ] Kindly make sure you stick to the design as much as possible. 
- [ ] Kindly make sure you use a similar typo(`font-size, weight and color`) as the design
Your design :point_down: 
image
Figma design :point_down: 
image

File: index.html [hunk 8]
```diff
@@ -68,6 +68,153 @@ <h1>
...
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang">
+          <li><a href="">HTML</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">JAVASCRIPT</a></li>
+        </ul>
```
Comment (christianbiring1) [inline index.html:105]: - [ ] Kindly make sure your stack list style is similar to the style in the Figma design template.(color, typo, spacing)
image
Figma design :point_down: 
image

File: index.html [hunk 9]
```diff
@@ -68,6 +68,153 @@ <h1>
...
+          <li><a href="">BACKEND DEV</a></li>
+          <li><a href="">2015</a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang">
+          <li><a href="">HTML</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">JAVASCRIPT</a></li>
+        </ul>
+        <button>See Project</button>
```
Comment (christianbiring1) [inline index.html:124]: - [ ] kindly make sure you fix this overlpping
Your design :point_down: 
Screenshot from 2022-10-31 11-06-00

Figma design :point_down: 
image

File: index.html [hunk 10]
```diff
@@ -68,6 +68,153 @@ <h1>
...
+      </div>
```
Comment (christianbiring1) [inline index.html:143]: - [ ] Kindly make sure your elements are well positioned on the page. Consider using the same spacing (`margin, padding`) as the design in Figma. :smile: 

Your design :point_down: 
image

Figma design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @QueenterJuma , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `summary`
- [ ] Good looking UI :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Updating your README and add the crucial points that are missing.
`Get started`
`That tell(shows how someone can make a copy of your repos and run it on their local computer)`
- [ ] Adding a detailed description telling what your project is about
- [ ] Giving your PR a descriptive `title`, `update index.html` doesn't link with the project.
`HINT: You can consider saying for ex: Finish mobile version` :smile: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 11]
```diff
@@ -48,6 +90,26 @@
...
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  display: flex;
+}
+
+.lang li a {
```
Comment (topeogunleye) [inline style.css:103]: - [ ] [OPTIONAL] I kindly suggest that you make sure your stack list style is similar to the style in the Figma design template.(color, typo, spacing). The colors you use here are overriding the correct colors from `.lang li` css selector

| Your Design  | Figma Design |
| ------------- | ------------- |
| image  | image  |

**Review by topeogunleye — APPROVED** [review]
### Hi @QueenterJuma, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* [ ]  Please check my inline comments.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**